dlme_airflow/drivers/oai_xml.py

```diff
@@ -65,10 +65,14 @@ def _from_metadata(self, manifest: etree) -> dict:
         NS = {'oai_dc': "http://www.openarchives.org/OAI/2.0/oai_dc/"}
         oai_block = manifest.xpath("//oai_dc:dc", namespaces=NS)[0]  # we want the first result
         for metadata in oai_block.getchildren():
+            logging.info(f"metadata is type: {type(metadata)} and value: {metadata}")
```

Comment:
I think we may want to start removing `logging.info` calls like this unless there is a good reason to maintain it in the running logs. 

Was this for local dev/debugging or do we have a reason for maintaining this logging in AWS?

Reply:
Yep, this should be pulled out. Its only for local debugging.